actuator/sim/robstride.py

```diff
@@ -0,0 +1,213 @@
+"""Simulated Robstride motors."""
+
+import time
+from enum import Enum, auto
+
+from actuator import RobstrideMotorControlParams, RobstrideMotorFeedback
+
+
+class MotorType(Enum):
+    Type01 = auto()
+    Type02 = auto()
+    Type03 = auto()
+    Type04 = auto()
+
+    @classmethod
+    def from_str(cls, s: str) -> "MotorType":
+        try:
+            return cls[f"Type{s}"]
+        except KeyError:
+            raise ValueError(f"Invalid motor type: {s}")
+
+
+class MotorConfig:
+    def __init__(
+        self,
+        p_min: float,
+        p_max: float,
+        v_min: float,
+        v_max: float,
+        kp_min: float,
+        kp_max: float,
+        kd_min: float,
+        kd_max: float,
+        t_min: float,
+        t_max: float,
+        zero_on_init: bool,
+        can_timeout_command: int,
+        can_timeout_factor: float,
+    ) -> None:
+        self.p_min = p_min
+        self.p_max = p_max
+        self.v_min = v_min
+        self.v_max = v_max
+        self.kp_min = kp_min
+        self.kp_max = kp_max
+        self.kd_min = kd_min
+        self.kd_max = kd_max
+        self.t_min = t_min
+        self.t_max = t_max
+        self.zero_on_init = zero_on_init
+        self.can_timeout_command = can_timeout_command
+        self.can_timeout_factor = can_timeout_factor
+
+
+ROBSTRIDE_CONFIGS = {
```

Comment:
can we keep constants in the rust side and expose them as python bindings, instead of putting them in python side? that way there is less duplication